src/libaktualizr-c/test/api-test-utils/api-test-utils.h

```diff
@@ -0,0 +1,36 @@
+#ifndef API_TEST_UTILS_H
+#define API_TEST_UTILS_H
+
+#ifdef __cplusplus
+#include "../../aktualizr/src/libaktualizr/config/config.h"
```

Comment:
We try to avoid these error-prone relative includes. config.h is directly accessible by `#include "config/config.h"`. For test_utils.h, it should be solveable by playing with CMake include directories for some test targets.

We can maybe define a CMake function in the spirit of `add_aktualizr_test`, dedicated for C tests, if needed. It can probably wait until we start having two or three of these, though.

Reply:
Fixed by CMake include directories.
add_aktualizr_test function is not added for now.